app/src/main/res/layout/fragment_home_screen.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context=".ui.overview.OverviewFragment">
+
+    <data class=".OverviewBinding">
+
+        <variable
+            name="viewModel"
+            type="com.example.hungrywolfes.ui.overview.OverviewViewModel" />
+
+        <variable
```

Comment:
This is a valid approach, however instead of passing the fragment as a variable, we can only use the viewmodel and use a `command` pattern to convey the navigation intents to the fragment. This is so you know in the future :D